code/game/objects/items/dice.dm

```diff
@@ -64,6 +64,9 @@
 	desc = "A die with four sides. The nerd's caltrop."
 	icon_state = "d4"
 	sides = 4
+	force = 3
+	embed_chance = 40

```

Comment:
40 is too much

Reply:
That's just the chance to actually embed though, nothing to do with damage

Reply:
Yeah I know